### PR TITLE
Fix/integration tests GitHub

### PR DESCRIPTION
**Pull request by cnlangzi**
- run blobber outside docker on integration-tests. and save time to build docker images. It takes only 2-5 minutes now. it was 20-30 minutes.
- add commands `make local-init` `make local-build` and `make local-run` on Makefile for local development and debugging
- fixed some bugs on grpc test code.

### CODE REVIEW

File: docker.local/b0docker-compose-github.yml [hunk 1]
```diff
@@ -26,26 +26,6 @@ services:
...
-      - ./keys_config:/blobber/keysconfig
-      - ./blobber${BLOBBER}/data/tmp:/tmp
-    ports:
-      - 505${BLOBBER}:505${BLOBBER}
-      - 3150${BLOBBER}:3150${BLOBBER}
-    command: ./bin/blobber --port 505${BLOBBER} --grpc_port 3150${BLOBBER} --hostname 198.18.0.9${BLOBBER} --deployment_mode 0 --keys_file keysconfig/b0bnode${BLOBBER}_keys.txt --files_dir /blobber/files --log_dir /blobber/log --db_dir /blobber/data --minio_file keysconfig/minio_config.txt
-    networks:
-      default:
-      testnet0:
-        ipv4_address: 198.18.0.9${BLOBBER}
```
Comment (kushthedude) [inline docker.local/b0docker-compose-github.yml:47]: Why did you remove this?

File: Makefile [hunk 2]
```diff
@@ -22,10 +24,44 @@ lint:
...
+	--port 5051 \
+	--grpc_port 31501 \
+	--hostname 127.0.0.1 \
+	--deployment_mode 0 \
+	--keys_file ../docker.local/keys_config/b0bnode1_keys.txt  \
+	--files_dir ./data/blobber/files \
+	--log_dir ./data/blobber/log \
+	--db_dir ./data/blobber/data  \
+	--minio_file ../docker.local/keys_config/minio_config.txt \
+	--config_dir ./data/blobber/config
```
Comment (kushthedude) [inline Makefile:63]: Nice 👍🏽

File: .github/workflows/tests.yml [hunk 3]
```diff
@@ -47,23 +42,29 @@ jobs:
...
         run: |
+          make local-build
+          sudo make local-run &
+          sleep 15
+          running=0
+          echo "checking blobber server status"
+          if [ $running -eq 0 ]; then
+            echo "checking blobber server status"
+            curl 127.0.0.1:5051 && $running=1 || sleep 2
+          fi
```
Comment (kushthedude) [inline .github/workflows/tests.yml:61]: I would prefer having a job to test the dockerised builds as this is what would actually get deployed on production.

File: docker.local/b0docker-compose-github.yml (hunk 1 above)
Comment (cnlangzi) [reply docker.local/b0docker-compose-github.yml:47]: it is implemented by `make local-run` instead .

Comment (cnlangzi) [reply docker.local/b0docker-compose-github.yml:47]: see https://github.com/0chain/blobber/blob/162c29b0f2b938160004354dfb770fdc7bcd235c/Makefile#L51

Comment (cnlangzi) [reply docker.local/b0docker-compose-github.yml:47]: only postgres and postgres-post are running inside docker. It is difficult to debug test code inside docker. so move them out from docker

File: .github/workflows/tests.yml (hunk 3 above)
Comment (cnlangzi) [reply .github/workflows/tests.yml:61]: No. it is not deployed on production . it is only working on github action. docker image is used on production.

Comment (cnlangzi) [reply .github/workflows/tests.yml:61]: And it is not a dockerize build. it is `go build` and `run local app` in background . so check port to verify http server if it is up. if not, tests code might not work.
